botx/bots/bots.py

```diff
@@ -132,12 +132,20 @@ async def execute_command(self, message: dict) -> None:
 
         await self(msg)
 
-    async def authorize(self) -> None:
+    async def authorize(self, *args: Any) -> None:
         """Process auth for each bot account."""
         for account in self.bot_accounts:
-            token = await self.get_token(
-                account.host,
-                account.bot_id,
-                account.signature,
-            )
+            try:
+                token = await self.get_token(
+                    account.host,
+                    account.bot_id,
+                    account.signature,
+                )
+            except exceptions.BotXConnectError:
+                logger.bind(botx_bot=True).warning(
+                    f"Credentials `host - {account.host}, "  # noqa: WPS305
+                    f"bot_id - {account.bot_id}` invalid.",
+                )
```

Comment:
I wanna know the reason of exception too. (nonexistent host/timeout/invalid secret)